docs/API.md

```diff
@@ -527,11 +514,7 @@ An oject with the following property:
 Returns the set of valid [*language tags*][Language tag].
 
 > Note: If there are two [*locale tags*][LANGUAGE.locale] in [LANGUAGE.locale]
-> that would give the same [short language tag][Language tag] then the short tag
-> is excluded from the set of valid [*language tags*][Language tag].
->
-> Note: Language tags that include country code are *deprecated* (planned
-> removal: v2023.1).
+> that would give the same *[language tag]* then is excluded.
```

Comment:
I’m not sure whether this paragraph still makes any sense after the edit. I don’t quite understand it myself.

Reply:
I removed it.